### PR TITLE
GH-821: replaced witness banner and section to ripple out

### PR DESCRIPTION
**Pull request by ksuhartono97**

### CODE REVIEW

**Review by yuenyauu — APPROVED** [review]

File: ui/src/components/ripple-out/RippleOutBanner.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+            <HStack h="100%" spacing="1">
+              <Box>
+                <Text
+                  position="relative"
+                  fontSize={['0.7em', 'xl']}
+                  as="i"
+                  color="#182E57"
+                  fontWeight={600}
+                >
+                  CLICK HERE TO LEARN MORE ABOUT THE RIPPLE OUT CAMPAIGN !
```
Comment (Midnite1212) [inline ui/src/components/ripple-out/RippleOutBanner.js:27]: Is the banner text supposed to be like that when there's 2 lines or centered ya?
image

File: ui/src/components/ripple-out/RippleOutSection.js [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+        <Container maxW="container.xl" h="100%">
+          <Center h="full" flexDir="column">
+            <Flex w="100%" flexDir="column" color="#ffffff">
+              <Text
+                textStyle="darker_grotesque_bold"
+                fontWeight="900"
+                fontSize={['4em', '14em']}
+                lineHeight={['1em', '0em']}
+              >
+                RIPPLE OUT
```
Comment (Midnite1212) [inline ui/src/components/ripple-out/RippleOutSection.js:31]: Same here, think there's not enough space if this becomes 2 lines on some devices.

image

**Review by Midnite1212 — CHANGES_REQUESTED** [review]

File: ui/src/components/ripple-out/RippleOutSection.js (hunk 2 above)
Comment (ksuhartono97) [reply ui/src/components/ripple-out/RippleOutSection.js:31]: i fixed, check again

File: ui/src/components/ripple-out/RippleOutBanner.js (hunk 1 above)
Comment (ksuhartono97) [reply ui/src/components/ripple-out/RippleOutBanner.js]: i fixed, check again

Comment (Midnite1212) [reply ui/src/components/ripple-out/RippleOutBanner.js]: lgtm :)

File: ui/src/components/ripple-out/RippleOutSection.js (hunk 2 above)
Comment (Midnite1212) [reply ui/src/components/ripple-out/RippleOutSection.js:31]: image
in ipads it became like this :")

**Review by Midnite1212 — APPROVED** [review]
Lgtm! :)

**Pull request merged**